modules/cluster-logging-loki-deploy.adoc

```diff
@@ -0,0 +1,132 @@
+// Module is included in the following assemblies:
+//cluster-logging-loki.adoc
+:_content-type: PROCEDURE
+[id="logging-loki-deploy_{context}"]
+= Deploying the LokiStack
+You can use the {product-title} web console to install the LokiOperator.
+
+.Prerequisites
+
+* {logging-title-uc} Operator 5.5
+* S3 bucket for log storage
```

Comment:
Should we mention somewhere that other object storages are implemented (see [here](https://loki-operator.dev/docs/object_storage.md/)) but currently "not supported" (= untested)?

Reply:
@periklis  @xperimental Will we support the other object storage beside of s3?   QE had verified the chunk be written into these storage.

Reply:
Documented = supported, generally speaking. So are they not supported as in we don't plan to support them, or considered "tech preview"?

Reply:
As far as I am aware we planned to take S3 to GA first and the follow-up with the other object storages later on. The implementation turned out to be pretty simple and has already happened and I was not aware that other object storages aside from the S3 / S3-compatible (ODF, Minio, ...) storages have already been tested. Also we might be talking of different things, for GCS I know that they _also_ have a S3-compatible API, but that's not what we are talking about when talking about using GCS as an object storage, but rather the "GCS native" API. Similar offers _might_ be available from the other providers as well.

For now, I think we could go with documenting only S3 and maybe linking to the loki-operator documentation for "further information", but my information might also be outdated.

Reply:
Got it. Thanks